modules/serverpod_auth/serverpod_auth_server/lib/src/business/config.dart

```diff
@@ -168,6 +181,8 @@ class AuthConfig {
     this.maxPasswordLength = 128,
     this.minPasswordLength = 8,
     this.allowUnsecureRandom = false,
+    this.generatePasswordHashCallback,
```

Comment:
Suggestion: Rename to `passwordHashGenerator` and `passwordHashValidator`.

This follows the pattern in the file for the rest of the callback methods, such as `userImageGenerator`.